cmake/package_settings.cmake

```diff
@@ -78,8 +77,14 @@ install(
   # Note that this is used in `openenclaverc` to set the path for
   # users of the SDK and so must remain consistent.
   DESTINATION ${CMAKE_INSTALL_LIBDIR}/openenclave/cmake
-  FILE openenclave-targets.cmake
-  COMPONENT OEHOSTVERIFY)
+  FILE openenclave-targets.cmake)
+install(
+  EXPORT openenclave-hostverify-targets
+  NAMESPACE openenclave::
+  DESTINATION ${CMAKE_INSTALL_LIBDIR}/openenclave/cmake
+  FILE openenclave-hostverify-targets.cmake
+  COMPONENT OEHOSTVERIFY
+  EXCLUDE_FROM_ALL)
```

Comment:
what is exclude_from_all?

Reply:
It means that this is only installed for the component, but not for the full OE package.